app/src/main/java/com/door43/translationstudio/newui/home/ImportDialog.java

```diff
@@ -232,6 +235,17 @@ private void importUri(Uri uri) {
         }
     }
 
+    private void applySourcesToProjects(Translator translator, String[] targetTranslationSlugs) {
```

Comment:
no it's quite small. We're already hitting the disk to load the translated frames anyway.